metriq_gym/benchmarks/qml_kernel.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from dataclasses import dataclass
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import ParameterVector
+from qiskit.circuit.library import unitary_overlap
+
+from qbraid import GateModelResultData, QuantumDevice, QuantumJob
+
+from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData, BenchmarkResult
+from metriq_gym.task_helpers import flatten_counts, flatten_job_ids
+
+
+@dataclass
+class QMLKernelData(BenchmarkData):
+    num_qubits: int
+
+
+@dataclass
+class QMLKernelResult(BenchmarkResult):
+    accuracy_score: float
+
+
+def ZZfeature_circuit(N: int) -> QuantumCircuit:
+    """Create a ZZ feature map in the same flavor as arXiv:2405.09724
+
+    Parameters:
+        N (int): Number of qubits
+
+    Returns:
+        QuantumCircuit: Desired circuit
```

Comment:
Might be good to add a bit more specific description to what the desired circuit is.

Reply:
Something for @Changhao-Li, I just copied this function from his notebook :)

Reply:
done